config.yml.example

```
@@ -3,6 +3,8 @@ server:
   loglevel: "info"
   url: "https://cal.julian-lemmerich.de"
   templatepath: /opt/ical-relay/templates
+  faviconpath: /static/media/favicon.svg
+  name: "Vorlesungsplan"
```

```suggestion
  name: "Calendar"
```
keep config.yml.example default in line with binary defaults